examples/apps/generic_client/generic_client.py

```diff
@@ -66,7 +66,8 @@ def ParseCommandLine(args) :
 		help="an address (hex string) of the smart contract (e.g. Worker registry listing)",
 		type=str)
 	parser.add_argument("-m", "--mode",
-		help="should be one of listing or registry",
+		help="should be one of listing or registry (default)",
```

Comment:
I found an answer on StackOverflow, can that be used here to reduce boilerplate code?
https://stackoverflow.com/questions/15836713/allowing-specific-values-for-an-argparse-argument

Reply:
Thanks.I added `               choices={"registry", "listing"}, for `--mode`